src/libraries/Common/src/Extensions/ParameterDefaultValue/ParameterDefaultValue.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Do you need this?

Reply:
It's a shared source file. So you have to annotate it in case other projects that reference it haven't been updated to nullable.